airbrakes/state.py

```diff
@@ -144,6 +144,9 @@ def update(self):
         # if our prediction is overshooting our target altitude, extend the airbrakes
         if self.context.apogee_predictor.apogee > TARGET_ALTITUDE:
             self.context.extend_airbrakes()
+        else:
```

Comment:
for the NASA launch we have to stay within a range of altitudes correct? e.g. 6000-6500ft?

Reply:
yes, from 4000-6000 ft, but we are trying to hit a specific target apogee inside of that range.

Reply:
yeah, that's what I was going to suggest. Our variable should be set maybe 60% in the range, so something like 5200ft.

Reply:
We have a specific height chosen by aubri. For the competition rn, we are thinking of 4800 ft